showcase/src/assets/translations/nl.json

```diff
@@ -18,6 +18,19 @@
         "PARENT": "Ouder",
         "CHILD": "Kind"
       },
+      "COLORS": {
+        "ADVANCED": {
+          "DETAILS": "Angular Material uses three color palettes: a primary palette, an accent palette and a warn palette. Stark gives you the ability to define those palettes through a SCSS map called <code>$stark-base-theme</code>. This map contains 3 properties, one for each Angular Material's palette and you need to use a SCSS function called <code>mat-palette()</code> in order to instantiate Angular Material theming the right way. Note that you need to import Angular Material's theming before using the <code>mat-palette()</code> function.",
+          "LINKS": "For more details on how Angular Material deals with theming, check out <a href=\"https://material.angular.io/guide/theming\" target=\"_blank\">https://material.angular.io/guide/theming</a><br>You can find the Material Design color palettes here: <a href=\"https://material.io/design/color/the-color-system.html#tools-for-picking-colors\" target=\"_blank\">https://material.io/design/color/the-color-system.html#tools-for-picking-colors</a><br>If you want to create your own palette, this tool can help you: <a href=\"http://mcg.mbitson.com/\" target=\"_blank\">http://mcg.mbitson.com/</a>",
+          "TITLE": "Advanced usage with color palettes"
+        },
+        "BASIC": {
+          "DETAILS": "All colors used in Stark can be set through a SCSS map called <code>$stark-color-theme</code>. Each color has a contrast property, which is needed to define the text color on top of this color. Once this map has been set, you need to import the Stark global theme to apply it.",
+          "EXAMPLE": "Here is a full version of the map is shown but you can also use it partially:",
+          "TITLE": "Basic usage"
+        },
+        "TITLE": "Stark Colors"
+      },
```

Comment:
Dutch translation 🎁:

"COLORS": {
  "ADVANCED": {
    "DETAILS": "Angular Material gebruikt drie kleurenpaletten: een primair palet, een accentpalet en een waarschuwingspalet. Stark biedt je de mogelijkheid om deze paletten te definiëren aan de hand van een SCSS map met de naam <code>$stark-base-theme</code>. Deze map bevat 3 eigenschappen, één voor elk Angular Material palet. Je moet de SCSS functie met de naam <code>mat-palette()</code> gebruiken om de Angular Material thematisering op correcte wijze te instantiëren. Merk op dat je het thema van Angular Materiaal moet importeren voordat je de functie <code>mat-palette()</code> gebruikt.",
    "LINKS": "Meer details over hoe Angular Material thematisering gebruikt, kan je terugvinden via volgende link <a href=\"https://material.angular.io/guide/theming\" target=\"_blank\">https://material.angular.io/guide/theming</a><br>Je kan de Material Design kleurenpaletten hier bekijken: <a href=\"https://material.io/design/color/the-color-system.html#tools-for-picking-colors\" target=\"_blank\">https://material.io/design/color/the-color-system.html#tools-for-picking-colors</a><br>Indien je je eigen palet wil ontwerpen, kan deze tool je hierbij helpen: <a href=\"http://mcg.mbitson.com/\" target=\"_blank\">http://mcg.mbitson.com/</a>",
    "TITLE": "Geavanceerd gebruik van kleurenpaletten"
  },
  "BASIC": {
    "DETAILS": "Alle kleuren die in Stark worden gebruikt, kunnen bepaald worden door het gebruik van een SCSS map met de naam <code>$stark-color-theme</code>. Elke kleur heeft een contrasteigenschap die nodig is om de tekstkleur bovenop deze kleur te definiëren. Nadat deze SCSS map is ingesteld, moet je het globaal thema van Stark importeren om het toe te passen.",
    "EXAMPLE": "Hier wordt een volledige versie van de SCSS map getoond, maar je kan die ook gedeeltelijk gebruiken:",
    "TITLE": "Basisgebruik"
  },
  "TITLE": "Stark Kleuren"
},

Reply:
Thanks!